### PR TITLE
Spec-related updates

### PR DESCRIPTION
**Pull request by jnpkrn**

### CODE REVIEW

File: pacemaker.spec.in [hunk 1]
```diff
@@ -568,13 +568,16 @@ getent group %{gname} >/dev/null || groupadd -r %{gname} -g 189
...
-
-%postun -n %{name}-libs -p /sbin/ldconfig
-
-%post -n %{name}-cluster-libs -p /sbin/ldconfig
+%if %{defined ldconfig_scriptlets}
+%ldconfig_scriptlets libs
+%ldconfig_scriptlets cluster-libs
+%else
+%post libs -p /sbin/ldconfig
+%postun libs -p /sbin/ldconfig
```
Comment (kgaillot) [inline pacemaker.spec.in]: I don't quite understand the syntax ...

Why did we originally call ldconfig for both library packages' %post but only -libs' %postun? Someone could install/uninstall just the -libs package, or just the -cluster-libs package (if -libs had already been installed), so I would think we would need ldconfig in all situations, to be safe.

Why drop ${name}-?

Why drop -n?

Comment (jnpkrn) [reply pacemaker.spec.in]: It's the same logic across dealing with subpackages with built-in
(directly in spec parser) or native macros, your subpackage
is either of the form:
- `%{name}-foo`, then it's customary to use `%SIGNATURE foo`, or
- `something-else`, then it's customary to use `%SIGNATURE -n something-else`

My guess is this had something to do with expressing that packagers
should preferably keep the subpackages properly namespaced as they
get rewarded with more terse syntax then
(indeed unless new use-case specific packaging conventions apply).

**Comment by jnpkrn** [comment]
Fixed "non-fatail" typo in commit message.

File: pacemaker.spec.in (hunk 1 above)
Comment (jnpkrn) [reply pacemaker.spec.in]: Re first question, there are all four variations in the original, perhaps a matter
of how github split the view?

**Comment by jnpkrn** [comment]
Anyway, no rush, I think I'll have at least one more change.

File: pacemaker.spec.in (hunk 1 above)
Comment (kgaillot) [reply pacemaker.spec.in]: Ah, I was looking at this too late in the evening :)

What's here looks good then, let me know when you're done

**Comment by jnpkrn** [comment]
The batch is complete, but let me test it a bit.

**Comment by jnpkrn** [comment]
Actually one more thing I wanted to check/change, related to
rpm support prior to
https://github.com/rpm-software-management/rpm/commit/517f7679008529f3552f1ef5ab2cb9a570ef8e0e

**Comment by jnpkrn** [comment]
The possibly problematic part was OK, I just clarified it a bit
(and consequently made it easier for Linux distributions with
based rpm>=4.14 to drop then-unnecessary burden).

And also surprising was that `-doc` was arch-specific, unnoticed
to this day :-)

**Comment by jnpkrn** [comment]
Added a commit not directly related to spec but discovered when testing rpmbuild.

**Comment by jnpkrn** [comment]
Went ahead and removed some more specfile cruft accumulated over time.

**Comment by jnpkrn** [comment]
As a cherry on the cake, I finally ditched the "timezone hack".

File: pacemaker.spec.in (hunk 1 above)
Comment (jnpkrn) [reply pacemaker.spec.in]: Added reference to documentation at the respective commit.

**Pull request merged**